<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Log_options</class>
 <widget class="QDialog" name="Log_options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Flags</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="cb_readOnly">
           <property name="text">
            <string>readOnly</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_acceptRichText">
           <property name="text">
            <string>acceptRichText</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_NoCRLF">
           <property name="text">
            <string>no CRLF</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_add_DateTime">
           <property name="text">
            <string>add DateTime</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_Color">
           <property name="text">
            <string>color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_ErrorAsMessage">
           <property name="text">
            <string>Error as Message</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_Text_Is_Windows">
           <property name="text">
            <string>is windows</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Codecs</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>CodecForCStrings</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cb_CodecForCStrings"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
